apps/RiiTag RPC/uninstall

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sudo rm /usr/local/bin/riitag-rpc
```

Comment:
use `sudo rm -f` so that a missing file will not cause the script to error (like if the user manually removed it or the install was unsuccessful